npm/webpack-dev-server/src/plugin.ts

```diff
@@ -10,9 +10,9 @@ interface CypressOptions {
 
 type CypressCTWebpackContext = {
   _cypress: CypressOptions
-} & webpack.compilation.Compilation
+} & compilation.Compilation
 
-export default class CypressCTOptionsPlugin implements webpack.WebpackPluginInstance {
+export default class CypressCTOptionsPlugin implements Plugin {
   private readonly files: any[] = []
```

Comment:
Cannot find `WebpackPluginInstance` type anywhere

Reply:
It's only on webpack 5...